src/main/java/com/pluralsight/CarDealership/CarDealershipApplication.java

```diff
@@ -1,13 +1,47 @@
 package com.pluralsight.CarDealership;
 
+import com.pluralsight.CarDealership.consoleapp.TableHeaders;
+import com.pluralsight.CarDealership.models.Vehicle;
+import com.pluralsight.CarDealership.models.VehicleType;
+import com.pluralsight.CarDealership.repositories.VehicleRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
+import java.util.List;
+import java.util.Scanner;
+
+import static com.pluralsight.CarDealership.models.VehicleType.*;
+
 @SpringBootApplication
-public class CarDealershipApplication {
+public class CarDealershipApplication implements CommandLineRunner {
+
+	@Autowired
+	private VehicleRepository vehicleRepository;
+	Scanner scanner = new Scanner(System.in);
 
 	public static void main(String[] args) {
 		SpringApplication.run(CarDealershipApplication.class, args);
 	}
 
+	@Override
+	public void run(String... args) throws Exception {
+		List<Vehicle> allVehicleList = vehicleRepository.getAllVehicles();
+		List<Vehicle> makeModelVehicleList = vehicleRepository.getVehiclesByMakeModel("toyota", "CAMRY");
```

Comment:
So the next step is to add to ask the user questions to ask them how they'd like to see their data. You can use a UserInterface class to hold that logic.